Dockerfile

```diff
@@ -1,6 +1,14 @@
+FROM golang:1.14.2-buster AS build
+
+WORKDIR /go/src/app
+
```

Comment:
If you are going to make use of `go.mod`, you can utilize it in Docker build steps to minimize pulls.

```
WORKDIR /go/src/app

## Download modules and store
COPY go.mod .
COPY go.sum .
RUN go mod download

COPY . .

RUN make build
```

In this manner, you don't have to download the modules every time you rebuild the container (making efficient use of docker image caching) unless you version bump them.